ynr/apps/moderation_queue/review_required_helper.py

```diff
@@ -0,0 +1,150 @@
+import abc
+from collections import namedtuple
+
+from django.conf import settings
+
+
+class BaseReviewRequiredDecider(metaclass=abc.ABCMeta):
```

Comment:
Nice. I like this approach. I don't have a strong opinion on this, but do you think calling these "Condition" objects would be more describtive than "Decider"? If you prefer "Decider", stick with that.

Reply:
I don't have strong opinions? Decider works well in my head, but, meh?